config.yaml.defaults

```diff
@@ -74,6 +74,8 @@ app:
       component: Filter
     - path: "/runs"
       component: ObservingRunPage
+    - path: "/allocations"
+      component: AllocationPage
```

Comment:
Probably want to put this below GCN events so it lines up with instruments and telescopes.

Reply:
done